README

```diff
@@ -32,4 +32,3 @@ Submit bug reports and feature requests to the PyYAML bug tracker:
 
 PyYAML is written by Kirill Simonov <xi@resolvent.net>.  It is released
 under the MIT license. See the file LICENSE for more details.
-
```

Comment:
removing trailing newlines is bad form as it's diff noise. I'd undo it for all the files you've added it for, which seems to be every file you've touched. Also files should end with a trailing newline to be POSIX valid